packages/network-controller/tests/NetworkController.test.ts

```diff
@@ -463,925 +463,858 @@ describe('NetworkController', () => {
       });
     });
 
-    for (const providerConfig of Object.values(NetworkType).map((networkType) =>
-      buildProviderConfig({ type: networkType }),
-    )) {
-      describe(`when the type in the provider configuration is "${providerConfig.type}"`, () => {
```

Comment:
The diff for this PR is ridiculous but it's really just one function being copy+pasted to a new function.

Try checking this branch out locally and replacing the new helper function with a copy of hte contents of this block on `main` to confirm; a diff should show that the only difference is the replacement of `controller.lookupNetwork()` with `operation(controller)`